src/widgets/angular-topics/ui/angular-topics.test.tsx

```diff
@@ -0,0 +1,27 @@
+import { render, screen } from '@testing-library/react';
+import { describe, expect, it } from 'vitest';
+import { AngularTopics } from './angular-topics';
+
+describe('AngularTopics component', () => {
+  it('renders without crashing', () => {
+    render(<AngularTopics />);
+    const angularTopics = screen.getByTestId('angular-topics');
+
+    expect(angularTopics).toBeVisible();
+  });
+
+  it('displays the title with correct text', () => {
+    render(<AngularTopics />);
+    const title = screen.getByTestId('widget-title');
+
+    expect(title).toBeVisible();
+    expect(title).toHaveTextContent('Topics Covered:');
+  });
+
+  it('displays topics list', () => {
```

Comment:
Maybe we should add test to check that list-content is displayed correctly?

Reply:
Done